### PR TITLE
more minor changes

### PR DESCRIPTION
**Pull request by slr**
to suppress #250. hope @ninedotnine wouldn't mind.

example

ha, I see what you did [there](https://github.com/notsecure/uTox/pull/1084).

---
This change is [Reviewable](https://reviewable.io/reviews/grayhatter/utox/267)

### CODE REVIEW

File: src/ui.c [hunk 1]
```diff
@@ -1103,31 +1113,52 @@ void ui_set_scale(uint8_t scale) {
...
             .type   = PANEL_DROPDOWN,
-            .x      = UTOX_SCALE(5   ),
-            .y      = UTOX_SCALE(15  ),
-            .height = UTOX_SCALE(12  ),
-            .width  = UTOX_SCALE(20  )
+            .x      = SCALE( 10),
+            .y      = SCALE( 30),
+            .height = SCALE( 24),
+            .width  = SCALE( 44)
         },
```
Comment (slr) [inline src/ui.c]: yep, `20 * 2 = 44`. because "Выкл" is a bit longer than "Off".

**Comment by GrayHatter** [comment]
So I don't have a reason to merge this yet. Is anyone planning to finish the beep on msg feature? 

---

Review status: 0 of 6 files reviewed at latest revision, 2 unresolved discussions.

---

*[src/ui.c, line 1126 \[r2\]](https://reviewable.io:443/reviews/grayhatter/utox/267#-KEK51hpbtnwbaupvZfR:-KEK51hpbtnwbaupvZfS:-260973185) ([raw file](https://github.com/grayhatter/utox/blob/6e4750ccca34f39b2eff4eb4efcb5175b96c19e1/src/ui.c#L1126)):*
this should be moved before every usage of UTOX_STR_WIDTH

---

*[src/ui.c, line 1134 \[r2\]](https://reviewable.io:443/reviews/grayhatter/utox/267#-KEK-S7znhA0TohxUA0E-r2-1134:-KEK4dUTaYTxXUyf8gGc:1315001356) ([raw file](https://github.com/grayhatter/utox/blob/6e4750ccca34f39b2eff4eb4efcb5175b96c19e1/src/ui.c#L1134)):*
UTOX_STR_WIDTH depends on setfont() being the same. see above

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/grayhatter/utox/267#-:-KEK5HRJyqo1IK1jZ0Ce:-1498734663)*

File: src/ui.c [hunk 2]
```diff
@@ -1103,31 +1113,52 @@ void ui_set_scale(uint8_t scale) {
...
+        /* save some before font changing */
+        int RINGTONE_STR_WIDTH      = UTOX_STR_WIDTH(RINGTONE);
+        int PUSH_TO_TALK_STR_WIDTH  = UTOX_STR_WIDTH(PUSH_TO_TALK);
+        #ifdef AUDIO_FILTERING
+        int BEEP_ON_MSG_STR_WIDTH   = UTOX_STR_WIDTH(BEEP_ON_MSG);
+        #endif
+        int IPV6_STR_WIDTH          = UTOX_STR_WIDTH(IPV6);
+        int UDP_STR_WIDTH           = UTOX_STR_WIDTH(UDP);
+        int CLOSE_TO_TRAY_STR_WIDTH = UTOX_STR_WIDTH(CLOSE_TO_TRAY);
+
```
Comment (slr) [inline src/ui.c]: > this should be moved before every usage of UTOX_STR_WIDTH

why? it's used only here.

File: src/ui.c (hunk 1 above)
Comment (slr) [inline src/ui.c]: > UTOX_STR_WIDTH depends on setfont() being the same. see above

I'm aware. it was just notification about intentional increasing of the width.

**Comment by slr** [comment]
> Is anyone planning to finish the beep on msg feature?

good question. I haven't noticed about the issue.

**Comment by ninedotnine** [comment]
If nobody beats me to it, I'm planning to take another look when I have more time.

**Comment by GrayHatter** [comment]
@ninedotnine if you do look back into you might want to track down @irungentoo on IRC he just wrote a bunch of music in str8c code, you might be able to get some help coming up with some cool beeps/tones for this

**Pull request closed without merging**